app/components/Upload/UploadImage.css

```diff
@@ -25,70 +13,92 @@
   width: inherit;
   z-index: 0;
   position: absolute;
-  background-color: gainsboro;
   cursor: pointer;
-}
 
-html[data-theme='dark'] .dropArea {
-  background-color: var(--color-mono-gray-1);
+  &:focus {
+    outline: none;
+  }
 }
 
 .inModalUpload {
-  height: 320px;
-  width: 600px;
+  width: 325px;
+  height: 200.861px;
+  border-radius: 1rem;
 }
 
 .placeholderContainer {
   overflow: hidden;
   border-radius: inherit;
   height: inherit;
-  display: flex;
-  flex-direction: column;
-  align-items: center;
-  justify-content: center;
   width: inherit;
   position: absolute;
 }
 
+.inModalUpload .placeholderContainer {
+  background-color: var(--color-gray-6);
+  border: 3px dashed var(--color-blue-6);
+}
+
+.focused .placeholderContainer {
+  border: 3px dashed var(--color-orange-6);
+}
```

Comment:
IMO having it default blue threw me off quite a bit as a blue border signifies focus (in FF and in chrome). I think having it just gray default and blue on focus is a better option.

Reply:
Well, yes but no. It's not like blue is the default of the web at everyone must abide by it. I think a very big problem with our current webapp is our use of gray and shades of it. Having everything be gray makes it look ugly, and we should be better at incorporating more colors where we can. I'm very much against having a gray border, as that will be a gray border around a slightly different gray box, around a slightly different gray bigger box on top of a gray background .. you see what I mean.

Reply:
A blue border is literally how FF and chrome shows focus unless overridden. So I would argue that most should not change it unless really required (i.e. pretty much never).

I do understand your point, but I also think that we should be careful with adding _too_ much color everywhere. I agree that a lot of color is nice, but with color shading in gray and white it can also look really nice. I might be more of a minimalist though so it is subjective. But, I'll use GitHub as an example here. They have a tint to their backgrounds and buttons, but still it is gray boxes with gray backgrounds and gray separator lines, gray pills etc. and it works really well. It's pleasant and not distracting. Color is nice, but don't overdo it.

Reply:
I see your point, but I don't feel like this is overdoing it. Also, I feel like this isn't just any normal border, but rather something to signify a drop area and something to interacted with, if that makes any sense, so I feel like a color is more greeting and therefore quite fitting.

I think it might look "too much" here because it's a screenshot which keeps the surrounding elements out. In reality, this modal is very small, and the color much less eye-catching together with a background with a bunch of stuff going on. Try it and see what you think. I think that might help.

Reply:
I tested this out, and it's worse than in the screenshot IMO. The effect that my mind immediately draws the conclusion that the box is selected is more apparent. I tried removing the border completely and it actually looks quite nice, maybe better I think. A slight box-shadow also works, but the cleaner  and simpler one is more appealing to me.

Reply:
I still prefer a dotted border. I think the dots signify that it's a "zone" better. What you show is just a very plain box which from a distance don't seem to have any purpose whatsoever. I don't see the whole argument of it looking "focused". Firstly, I don't think so. Secondly, how does that even matter?

Reply:
It's not like Google never adds a blue border to anything because that's reserved? Just as a quick example, see the document I sent earlier.